app/src/main/java/com/cornellappdev/score/components/UpcomingGameCard.kt

```diff
@@ -0,0 +1,161 @@
+import androidx.compose.material3.Card
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.score.theme.poppinsFamily
+import com.example.score.R
+
+@Composable
+fun UpcomingGameHeader(
+    leftTeamLogo: Painter,
+    rightTeamLogo: Painter,
+    gradientColor1: Color,
+    gradientColor2: Color ,
+    modifier: Modifier
+) {
+    Box(
+        modifier = modifier
+            .clip(RoundedCornerShape(topStart = 16.dp, topEnd = 16.dp))
+            .fillMaxWidth()
+            .background(
+                brush = Brush.linearGradient(
+                    colors = listOf(gradientColor1, gradientColor2)
+                )
+            )
+    ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.spacedBy(32.dp, Alignment.CenterHorizontally),
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(vertical = 36.dp)
+        ) {
+            Image(
+                painter = leftTeamLogo,
+                contentDescription = "Left Team Logo",
+                contentScale = ContentScale.FillBounds,
+                modifier = Modifier.size(60.dp)
+            )
+
+            Text(
+                text = "VS",
+                style = TextStyle(
+                    fontSize = 24.sp,
+                    fontFamily = poppinsFamily,
+                    fontWeight = FontWeight(700),
+                    fontStyle = FontStyle.Italic,
+                    color = Color(0xFFFFFFFF),
+                )
+            )
+
+            Image(
+                painter = rightTeamLogo,
+                contentDescription = "Right Team Logo",
+                modifier = Modifier.height(62.dp)
+            )
+        }
+    }
+}
+
+@Preview
+@Composable
+fun UpcomingGameHeaderPreview() {
+    UpcomingGameHeader(
+        leftTeamLogo = painterResource(R.drawable.cornell_logo),
+        rightTeamLogo = painterResource(R.drawable.penn_logo),
+        gradientColor1 = Color(0xE1A69F),
+        gradientColor2 = Color(0x011F5B) ,
+        modifier = Modifier
+    )
+}
```

Comment:
For future reference, if you have some private composable function just for the sake of organization, there's no need to write a Preview function for it (unless you want to; i do this sometimes to help debug.)